### DISCUSSION TITLE
fzf: open with trouble

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#3636] [General]
How can we open the selected stuffs on trouble?
https://github.com/LazyVim/LazyVim/blob/7bebc40333c59660b6bff7ac67bff510aecf4fb2/lua/lazyvim/plugins/extras/editor/telescope.lua#L177

This was possible on telescope

**Comment by folke** [comment]
have you even tried? That already works, same keymap

**Reply by rubiin** [reply]
Looks like some issues with my configs. Works on fresh install
